### DISCUSSION TITLE
Sitemap with private ip link

### DISCUSSION THREAD

**Discussion by hautph** [squidfunk/mkdocs-material#5762] [Q&A]
Hi,

I got a problem with sitemap, I config site_url in mkdocs.yml, but I got a problem when connecting to sitemap.xml, it displays ip_local:8000, and the source from the site is site_url

Please help me with how to config it.

**Comment by squidfunk** [comment]
This is essentially a MkDocs question. MkDocs will replace your URL with localhost when you invoke `mkdocs serve`, but when you run `mkdocs build`, the value you configured in `site_url` is used. If there's further unclarity, please ask the maintainers of MkDocs.
